src/main/kotlin/calculator/Convertor.kt

```diff
@@ -0,0 +1,22 @@
+package calculator
+
+class Convertor {
```

Comment:
클래스로 선언한다면 클래스가 자신의 프로퍼티를 가지고 프로퍼티가 메서드에서 사용되어야 의미가 있을것 같아요~!
Converter라는 클래스를 차라리 `식`을 나타내는 클래스로 명명이 되고 프로퍼티로 입력받은 식을 가지고 있고
그 클래스에게 숫자를 내어 놓아라, 연산자를 내어놓아라 와 같은 메세지를 던질수있는 방법도 있을것 같아요!

지금 같이 단순히 메서드만 가지고 있다면 Converter를 인스턴스화 할 필요가 있을까? 의문이 드네요!
참고로 이 처럼 단순히 메서드만 가지고 있고 상태가 없는 클래스를 활용하실 때는 코틀린에서는 아래와 같이 사용하는 방법도 있습니다~!
```suggestion
object Convertor {
```


[kotlin object class](https://kotlinlang.org/docs/object-declarations.html#object-declarations-overview)